### DISCUSSION TITLE
Cannot control the fan

### DISCUSSION THREAD

**Discussion by Higedann** [artisan-roaster-scope/artisan#852] [Q&A]
Hello, I'm a beginner for this software, and think it is really helpful with my roasting, Thank you Artisan team!

I connected PID successfully, but when I try to connect with fan it always no reaction. I'm using RS-485 translator to connect with PID, there are 2 points of junction on it, and I connect a 4-lines fan. What should I do to make the button or slider to control this fan? Thank you!

**Comment by bigl00z3** [comment]
Hello Higedann,

you will need to provide more data if you want other users to be able to help:

- What type of equipment / model your roaster is. You can post a link to the technical data/specifications if you have it.
- What type of PID / model you are using (I imagine a model communicating in MODBUS over the RS485 interface)
- What type of fan, how is it controlled, are you directly connecting to the fan? Or also wia the RS-485)

You can eventually upload an image of your connections /wirings.

Also please attach your current Artisan settings file (as exported via menu Help >> Save Settings as *.aset) file.
Please attach any relevant Artisan *.alog profiles.

_Note that you need either add a `.txt` extension or zip the files before uploading. Otherwise you will receive a "Not a supported file type" error on uploading._

Basicaly, the clearer you are about your questions, the better the answers will be.

For the buttons and sliders, have a look at the [https://artisan-scope.org/docs/quick-start-guide/](https://artisan-scope.org/docs/quick-start-guide/) documentation pages more particularly at the [Events](https://artisan-scope.org/docs/events/) page

Nico

**Reply by Higedann** [reply]
The problem has been solved, my Modbus command wasn't correct, thanks for your reply!
